### DISCUSSION TITLE
Whitelabel Dashboard

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#152] [Anything Else]
"Could dashboards be whitelabelled? Or perhaps just the dashboard shown via share links? It could be useful for freelancers and agencies sharing a dashboard link with clients.

I think something that allows logo and branding colors would be cool, so clients can login and get their own view of their reports!"

**Update:** Here's how our whitelabeling works https://plausible.io/white-label-web-analytics

**Comment by MoryCorp** [comment]
Something like this ? (In the website dashboard)

![image](https://user-images.githubusercontent.com/26336203/97233935-1fe2c300-17e0-11eb-8770-6e31668c35aa.png)
And if custom image is uploaded then disable footer/docs button ?

**Reply by cmdaltctr** [reply]
Hi, I just deployed Plausible for the first time with Caprover. I noticed that upon login with admin, I see the footer in the deployed web-app, just like in the website. Is this normal/ expected? Also I cannot add new user. Are these the limitations of the self-hosted version?

**Reply by metmarkosaric** [reply]
hi @AizatHawari! that's normal and expected. there's no way to remove Plausible branding from Plausible. 

about adding new users: Self-hosted is a long term release so latest features won’t be immediately available as they’re battled-tested in the cloud before released to self-hosted. We release self-hosted approximately twice per year. See https://plausible.io/self-hosted-web-analytics

**Comment by metmarkosaric** [comment]
Adding a note here for those that find this thread as we have some options now that companies use to integrate Plausible stats into their UI without any Plausible branding:

- [Embed dashboard](https://plausible.io/docs/embed-dashboard) feature allows you to embed our default dashboard (without Plausible branding) in your UI. You can even embed a specific filtered dashboard. This is the easy and quick option.

- [Stats API](https://plausible.io/docs/stats-api) feature allows you to take individual metrics and create a custom-built dashboard within your UI in whichever shape or form you want. This requires more development time from your side but you get full flexibility. No Plausible branding either.
